src/commands/referendum/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Client, Message, User } from "discord.js";
+import _ from "lodash";
+import { Command } from "../command.interface";
+import { hasVoted, isPoll } from "./helper";
+import {ReferendumCommandKeyList} from "./referendum.interface";
+// only one poll at a time
+var voted: string[] = [];
```

Comment:
should export this to the interface file so we can documented them later

Reply:
export what?

Reply:
I mean if it just array of primitive (`string`,`number`,etc) it's find but if you assign object as annotation, I think it's better if you export into an interface 
```
export interface VoteOption {
	[key: string]: number
}
```
also another thing is to use const and let instead of var